### DISCUSSION TITLE
how to add russian characters in regex?

### DISCUSSION THREAD

**Discussion by galaxyandrew** [TownyAdvanced/Towny#5509] [Q&A] [answered]
I have a command / t set board (text) and if this text is in Russian it will write that the wrong characters tell me what to write in the regex:
    name_filter_regex: '[\\\ /]'
    name_check_regex: ^ [\ p {L} a-zA-Z0-9 ._ \ [\] -] * $
    string_check_regex: ^ [a-zA-Z0-9 \ s ._ \ [\] \ # \? \! \ @ \ $ \% \ ^ \ & \ * \ - \, \ * \ (\) \ {\ }] * $
    name_remove_regex: '[^ \ P {M} a-zA-Z0-
nine\&._\[\]-]'

**Comment by LlmDl** [comment, marked as answer]
`\p{L}` needs to be in the string_check_regex.
